build-system/tasks/dist.js

```diff
@@ -66,10 +66,9 @@ const deglob = require('globs-to-files');
 function transferSrcsToTempDir() {
   const tmp = `${tempy.directory()}/amphtml`;
   process.env.AMP_TMP_DIR = tmp;
-  log(green(`creating temp directory: ${tmp}`));
 
   if (!isTravisBuild()) {
-    log('Writing transforms to', colors.cyan(tmp));
+    log('Transforming JS files to', cyan(tmp));
   }
   const files = deglob.sync(BABEL_SRC_GLOBS);
   log(green('Executing babel transforms'));
```

Comment:
Remove this line, since it duplicates line 71.

Reply:
done.